presto-docs/src/main/sphinx/release/release-318.rst

```diff
@@ -0,0 +1,58 @@
+===========
+Release 317
+===========
+
+General Changes
+---------------
+
+* Fix ``DISTINCT FROM`` for ``UUID`` type. (:issue:`1180`)
+* Improve performance for queries over ``information_schema`` that have a limit. (:issue:`999`)
+* Improve query performance when ``optimize_hash_generation`` is enabled. (:issue:`1071`)
+* Add support for ``Asia/Qostanay`` time zone. (:issue:`1221`)
+* Add session properties for ``query_max_memory_per_node`` and
+  ``query_max_total_memory_per_node``. (:issue:`1212`)
+* Rename Kerberos properties to make them consistent with other
+  HTTP server properties. (:issue:`1270`)
+* Introduce Google Sheets Connector. (:issue:`1030`)
+* Change query CPU tracking for resource groups to update periodically while
+  the query is running. Previously, CPU usage would only update at query
+  completion. This improves resource management fairness when using
+  CPU-limited resource groups. (:issue:`1128`)
+* Add :func:`last_day_of_month` function. (:issue:`1295`)
+* Remove ``distributed_planning_time_ms`` column from ``system.runtime.queries``. (:issue:`1084`)
+* Change the semantics of ``analysis_time_ms`` column from ``system.runtime.queries``.
+  Previously it was showing the time spent on query planning, now it  shows time spent
+  on semantic analysis. of query. (:issue:`1084`)
+* Add ``planning_time_ms `` column to ``system.runtime.queries`` that shows time
+  spent on query planning. (:issue:`1084`)
+
+Hive Connector Changes
+----------------------
+
+* Fix query failure due to missing credentials while writing empty bucket files. (:issue:`1298`)
+* Improve performance of S3 Object listing. (:issue:`1232`)
+* Improve performance when reading data from GCS. (:issue:`1200`)
+* Add support for reading data from S3 Requester Pays buckets. This can be enabled
+  using the ``hive.s3.requester-pays.enabled`` configuration property. (:issue:`1241`)
+
+Kafka Connector Changes
+-----------------------
+
+* Update Kafka client library to ``1.1.1``. (:issue:`1112`)
+
+PostgreSQL Connector Changes
```

Comment:
> The PostgreSQL items need to be replicated for MySQL, etc.

Sometimes we would write this as `MySQL, PostgreSQL, Redshift, and SQL Server Changes`